Lib/logging/__init__.py

```diff
@@ -191,10 +191,11 @@ def _checkLevel(level):
         rv = level
     elif str(level) == level:
         if level not in _nameToLevel:
-            raise ValueError("Unknown level: %r" % level)
+            raise ValueError(f"Unknown level: {repr(level)}")
         rv = _nameToLevel[level]
     else:
-        raise TypeError("Level not an integer or a valid string: %r" % level)
+        raise TypeError("Level not an integer or a valid string: "
```

Comment:
This could just be `"Level not an integer or a valid string: %s" % repr(level)` which should always work. I'm not against f-strings, but there is no need to introduce them here for this purpose.

Reply:
I went with `"...: %r" % (level,)"` since it's closer to the old code, but happy to push a commit to go with your variant if you want